<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 158, 158);</string>
  </property>
  <widget class="QPushButton" name="signup_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 83, 83);

</string>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 83, 83);</string>
   </property>
   <property name="text">
    <string>Have an account?</string>
   </property>
  </widget>
  <widget class="QLabel" name="random_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="random_line_edit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_line_edit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);
border-radius:11px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line_edit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);
color: rgb(36, 36, 36);
border-radius:11px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Enter the code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/lock.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/user.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
